### PR TITLE
[DEXC-2125] Updated packages and related changes

### PR DESCRIPTION
**Pull request by babshev**
Updated packages and completed required changes

### CODE REVIEW

File: test/SampleConnectorUnitTests/SampleConnectorUnitTests.csproj
Comment (rinku-thakur) [inline test/SampleConnectorUnitTests/SampleConnectorUnitTests.csproj:1]: @babshev , could you also change the path to GeometryPrimitives and GeometryUtilities in this unit test project reference. It is referring to 3.3.1 folder.

Comment (babshev) [reply test/SampleConnectorUnitTests/SampleConnectorUnitTests.csproj:1]: @rinku-thakur Thanks for comment. I have changed the path to GeometryPrimitives and GeometryUtilities in this unit test project reference. Now it's referring to the 4.0.3-beta folder.

File: test/SampleConnectorUnitTests/SampleConnectorUnitTests.csproj [hunk 1]
```diff
@@ -57,67 +57,65 @@
...
       <HintPath>..\..\src\packages\Autodesk.Extensions.Http.ForgeRetry.8.0.0\lib\netstandard2.0\Autodesk.Extensions.Http.ForgeRetry.dll</HintPath>
     </Reference>
     <Reference Include="Autodesk.Forge, Version=1.9.8.0, Culture=neutral, processorArchitecture=MSIL">
       <HintPath>..\..\src\packages\Autodesk.Forge.1.9.8\lib\net48\Autodesk.Forge.dll</HintPath>
     </Reference>
-    <Reference Include="Autodesk.GeometryPrimitives, Version=0.5.7.0, Culture=neutral, processorArchitecture=AMD64">
-      <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\..\src\packages\Autodesk.DataExchange.3.3.1-beta\build\native\libs\Autodesk.GeometryPrimitives.dll</HintPath>
+    <Reference Include="Autodesk.GeometryPrimitives">
+      <HintPath>..\..\src\packages\Autodesk.DataExchange.4.0.3-beta\build\native\libs\Autodesk.GeometryPrimitives.dll</HintPath>
```
Comment (rinku-thakur) [inline test/SampleConnectorUnitTests/SampleConnectorUnitTests.csproj:115]: @babshev , can we update the geometry packages here to refer from 4.0.4-beta folder?

Comment (babshev) [reply test/SampleConnectorUnitTests/SampleConnectorUnitTests.csproj]: Thanks @rinku-thakur. I have changed to 4.0.4-beta

**Pull request merged**